### PR TITLE
[Docs] Camera API docs refresh

### PR DESCRIPTION
**Pull request by dotNetkow**
- Update Angular code example - remove unnecessary code
- Add link to Ionic Angular camera guide
- Mention that for Android, CAMERA permission is actually not required (via Julio), so as to avoid confusion.
- Small formatting changes

### CODE REVIEW

File: site/docs-md/apis/camera/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This API requires the following permissions be added to your `AndroidManifest.xm
 <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 ```
 
-The first permission is for Camera access, and the storage permissions are for reading/saving photo files.
+The storage permissions are for reading/saving photo files. Note that `android.permission.CAMERA` is not required.
```
Comment (jcesarmobile) [inline site/docs-md/apis/camera/index.md]: I removed the CAMERA permission the other day, so not sure if this note makes sense

File: site/docs-md/apis/camera/index.md [hunk 2]
```diff
@@ -88,8 +89,8 @@ export class CameraPage {
       source: CameraSource.Camera
     });
     
-    // Example of using the Base64 return type. It's recommended to use CameraResultType.Uri
-    // instead for performance reasons when showing large, or a large amount of images.
+    // Recommended to use CameraResultType.DataUrl instead of Base64 for
```
Comment (jcesarmobile) [inline site/docs-md/apis/camera/index.md]: DataUrl is as bad as Base64, recommended is Uri

File: site/docs-md/apis/camera/index.md (hunk 1 above)
Comment (dotNetkow) [reply site/docs-md/apis/camera/index.md]: added it on the idea that people will be surprised that Camera permission isn't needed. I will remove

**Comment by dotNetkow** [comment]
> Added two comments.
> But does it make sense to keep the Angular example if we have the guide that uses Angular as it's more complete?

Good point, updated!

File: site/docs-md/apis/camera/index.md (hunk 2 above)
Comment (dotNetkow) [reply site/docs-md/apis/camera/index.md]: Updated

File: site/docs-md/apis/camera/index.md (hunk 1 above)
Comment (dotNetkow) [reply site/docs-md/apis/camera/index.md]: Maybe it would confuse them after all :)

Comment (jcesarmobile) [reply site/docs-md/apis/camera/index.md]: Yeah, I forgot to remove that part, so that part of your change was OK, just not the note

**Pull request merged**